cirrus/google_cloud/manager.py

```diff
@@ -314,7 +314,7 @@ def get_primary_service_account(self, proxy_group_id):
         if service_account_id_for_user in service_account_emails:
             primary_email = service_account_emails[service_account_id_for_user]
 
-        return self.get_service_account(primary_email)
+        return self.get_service_account(primary_email).json()
```

Comment:
These are breaking changes. did you make sure you fixed every spot where these functions gets called in fence?

Reply:
this refactoring may be better off waiting until we've done more implementation, since it might be a headache to fix all the spots in fence that rely on this. I'd almost suggest we wait to do this cirrus cleanup/refactor until later

Reply:
I changed all the code called to that function